src/main/java/com/rapid7/client/dcerpc/msvcctl/ServiceControlManagerService.java

```diff
@@ -71,41 +71,43 @@ public boolean startService(String service, ContextHandle serviceManagerHandle)
         return (response.getReturnValue().is(SystemErrorCode.ERROR_SUCCESS.getErrorCode()));
     }
 
-    public RQueryServiceStatusInfo queryService(String service, ContextHandle serviceManagerHandle)
+    public ServiceStatusInfo queryService(String service, ContextHandle serviceManagerHandle)
         throws IOException {
         ContextHandle serviceHandle = getServiceHandle(service, serviceManagerHandle);
         RQueryServiceStatusRequest request = new RQueryServiceStatusRequest(serviceHandle);
         RQueryServiceStatusResponse response = transport.call(request);
         if (response.getReturnCode().is(SystemErrorCode.ERROR_SUCCESS.getErrorCode())){
-            return response.getrQueryServiceStatusInfo();
+            return response.getServiceStatusInfo();
         } else return null;
     }
 
-    public RQueryServiceStatusInfo stopService(String service, ContextHandle serviceManagerHandle)
+    public ServiceStatusInfo stopService(String service, ContextHandle serviceManagerHandle)
```

Comment:
Should we return boolean instead? Does caller need this?

Reply:
Legacy code returned the service status upon starting and stopping the services. I tried to maintain this behaviour. Initial code was returning a boolean, but now we run an addition query to see whether the status has changed.

Reply:
Okay